### PR TITLE
Removed getDerivedState -> componentDidUpdate

### PR DESCRIPTION
**Pull request by ptomasroos**
Tries to resolve the issue brought in #155

### CODE REVIEW

**Comment by ptomasroos** [comment]
@FrikkieSnyman does this resolve your use-case mentioned

File: MultiSlider.js [hunk 1]
```diff
@@ -385,6 +327,48 @@ export default class MultiSlider extends React.Component {
...
+      return;
+    }
+
+    let nextState = {};
+    if (
+      prevProps.min !== this.props.min ||
+      prevProps.max !== this.props.max ||
+      prevProps.step !== this.props.step ||
+      prevProps.values[0] !== this.state.valueOne ||
+      prevProps.sliderLength !== this.props.sliderLength ||
```
Comment (FrikkieSnyman) [inline MultiSlider.js]: Should this not also compare to this.props.values[0]?

Comment (ptomasroos) [reply MultiSlider.js]: Good call sounds better indeed

**Comment by ptomasroos** [comment]
Can you try the branch out @FrikkieSnyman so I can move on and release the changes
Thanks !

**Comment by FrikkieSnyman** [comment]
Sorry, busy week! Will try get to it this week!

**Comment by ptomasroos** [comment]
Thanks !

**Comment by FrikkieSnyman** [comment]
Just tried it out, looks good! Thanks!

**Comment by ptomasroos** [comment]
Thanks for helping out! Will make a release

**Pull request merged**